sktime_dl/deeplearning/base/estimators/_classifier.py

```diff
@@ -43,14 +45,10 @@ def predict_proba(self, X, input_checks=True, **kwargs):
         Find probability estimates for each class for all cases in X.
         Parameters
         ----------
-        X : array-like or sparse matrix of shape = [n_instances, n_columns]
-            The training input samples.
-            If a Pandas data frame is passed (sktime format)
-            If a Pandas data frame is passed, a check is performed that it
-            only has one column.
-            If not, an exception is thrown, since this classifier does not
-            yet have
-            multivariate capability.
+        X : a nested pd.Dataframe, or array-like of shape =
+        (n_instances, series_length, n_dimensions)
```

Comment:
Isn't is `(n_instances, n_dimensions, series_length)`? I think that's the output from `nested_to_3d_numpy()`.

Reply:
You're right that's the output of that, however we ultimately need it in that transposed state for consistency 

https://github.com/sktime/sktime-dl/pull/63/files#diff-eadf622a84d7ec0f0ca9c4996ccdd5d4R102